### ISSUE TITLE
[Feature Request] Delete ROM action

### ISSUE THREAD

**Issue by Jocala** [Ryujinx/Ryujinx#6952]
### Overview

I'd like to see a "delete rom" feature added to the right-click menu in addition to the delete dlc/updates feature already present. This basic file management would be a very handy feature. Thanks!

### Smaller details

I'm unsure how this would be implemented.  A checkbox?

### Nature of request

An addition to current file management features.

### Why would this feature be useful?

As a ROM collection grows in size, being able to prune unwanted titles directly from the Ryujinx UI would be a very handy feature.

**Comment by MelonSpeedruns** [comment]
With the amount of people that already click on every setting not knowing what it does, having an option like that will 1000% lead to people randomnly deleting their files and then complain about it, or don't know where the file went.

**Comment by Jocala** [comment]
I disagree.  Put in a yes/no confirmation for sure, but file management is a feature not a bug. Are there complaints about deleting DLC/updates?  Anyway, I hope the developers give the request due consideration.

**Comment by gdkchan** [comment]
I don't think we have an option to "delete DLC/updates". We only have an option to remove them from the emulator, but it does not delete the DLC and update NSP files.
